MathlibTest/Deriv.lean

```diff
@@ -0,0 +1,25 @@
+import Mathlib
+
+/-! Test that `simp` can prove some lemmas about derivatives. -/
+
+open Real
+
+example (x : ℝ) : deriv (λ x => cos x + 2 * sin x) x = -sin x + 2 * cos x := by
+  simp
+
+example (x : ℝ) :
+    deriv (fun x ↦ cos (sin x) * exp x) x = (cos (sin x) - sin (sin x) * cos x) * exp x := by
+  simp; ring
+
+/- for more complicated examples (with more nested functions) you need to increase the
+`maxDischargeDepth`. -/
+
+example (x : ℝ) :
+    deriv (fun x ↦ sin (sin (sin x)) + sin x) x =
+    cos (sin (sin x)) * (cos (sin x) * cos x) + cos x := by
+  simp (config := {maxDischargeDepth := 3})
```

Comment:
Can you use the new +option syntax here? Also below?

Reply:
I had to dig to the actual PR description https://github.com/leanprover/lean4/pull/5883 to figure out that this had shorter syntax.